docs/Hardening.md

```diff
@@ -0,0 +1,197 @@
+# Linux Hardening while using OMS Log Collector
+We support some industry standard hardening (please see [Supported Hardening or Customization](#supported-hardening-or-customization) for more details).
+We also can provide some [troubleshooting steps](#common-hardening-issues) for customized hardening.
```

Comment:
Should we specify our support boundary here? e.g. does a customer who has done custom hardening who has followed this TSG and still been unable to get OMS to work qualify for support or not? Or are we saying FIPS and Cylance are OK, but the rest of this is unsupported (but here's some tips anyways)? @Lukeout

Reply:
I was trying to imply the latter with this, but I don't really know of a good way to word it that doesn't seem blunt

Reply:
Blunt is fine ... the document should not leave room for interpretation

Reply:
Changed here to be clearer:
https://github.com/microsoft/OMS-Agent-for-Linux/pull/1269/commits/b6a96c6bef0fd670a3c6676c17026d5228135af5#diff-ff8800189743d6567bcda8a83de5b22c943b28c51e4488cce0ea2b185716cc37R2